samples/boards/st/uart/index.rst

```diff
@@ -0,0 +1,5 @@
+.. zephyr:code-sample-category:: stm32_uart
+   :name: UART
+   :show-listing:
```

Comment:
Add `   :glob: */*`? 
There are some RST files further in this directory tree

Reply:
> Add ` :glob: */*`? There are some RST files further in this directory tree

Yes but they're picked up by other code sample categories further down -- this is now basically assembling everything one level at a time, to avoid double inclusion. 
If there were orphans after this change the build would actually fail!

Reply:
ok, thanks.